indra/newview/scripts/lua/test_LLChat.lua

```diff
@@ -0,0 +1,18 @@
+LLChat = require 'LLChat'
+
+function generateRandomWord(length)
+    local alphabet = "abcdefghijklmnopqrstuvwxyz"
+    local word = ""
+    for i = 1, length do
+        local randomIndex = math.random(1, #alphabet)
+        word = word .. alphabet:sub(randomIndex, randomIndex)
```

Comment:
I realize it doesn't matter for a test script, but when building up a string iteratively, I usually think in terms of appending to a list and then (in Lua) using `table.concat()` to construct the ultimate string.

Reply:
Sure, why not, let's show 'table' approach too.